.github/workflows/pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Reckon it's better to do v18, v20, and v22 (if v22 doesn't work we probably file a ticket and resolve it)